spacy_transformers/data_classes.py

```diff
@@ -99,15 +99,17 @@ def from_batch_encoding(cls, token_data: BatchEncoding) -> "WordpieceBatch":
             for tokens in token_data["input_texts"]
         ]
 
+        # The following tensors are intentionally allocated on the CPU to reduce
+        # host-to-device copies.
         numpy_ops = NumpyOps()
 
         return cls(
             strings=token_data["input_texts"],
-            input_ids=numpy_ops.asarray2i(token_data["input_ids"]),
+            input_ids=numpy_ops.asarray(token_data["input_ids"], dtype="int64"),
```

Comment:
I would refer to the dtype rather than hardcoding it? (Is it actually `int64`?)

Reply:
Good point. It is indeed `int64`, but I'll pass the source's `dtype` to `asarray`.